youtube_transcript_api/_api.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, video_id):
             self.video_id = video_id
 
     @classmethod
-    def get_transcripts(cls, video_ids, languages=None, continue_after_error=False, proxies=None):
+    def get_transcripts(cls, video_ids, languages=['en'], continue_after_error=False, proxies=None):
```

Comment:
Since this is statically initialized, all calls to `get_transcripts` share the same `languages`. Therefore it should probably be a tuple instead of list, as they are immutable.

Reply:
I get why you would want to make them tuples. The reason why I did this was because I didn't want the `self.languages` of a  `_TranscriptFetcher()` to sometimes be a list and sometimes a tuple,  as I feel that could be potentially confusing in the future.

Reply:
If this was statically typed, I would declare `languages` as an `Iterable<str>`. Both list and tuples implement this interface, therefore I think this shouldn't be a problem.

Reply:
I actually think that the issue with declaring a list as a default argument is not that it will use unnecessary memory for every function call, but that the list is actually the same list for all calls of the function. Therefore, its not really a good thing to do.

Reply:
That is exactly what I am saying! 😊 
Since it is statically initialized, all calls to this method use the same list. However, this is not a problem if `languages` is a tuple, since tuples are immutable. Memory is no issue here.